docs/docs/labelling-issues.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Labelling issues
+---
+
+## What are issue labels?
+
+Issue labels are a tool in GitHub that that is used to group issues into one or more categories.
+
+[Check out Gatsby's labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels)
+
+
+## Why label issues?
+
+Gatsby is a very active project with many new issues opened each day. Labelling issues helps by identifying:
+
+- good issues for new contributors to work on
+- reported and confirmed bugs
+- new features
+- duplicate issues
+- issues that are stalled or blocked
+
+## Who can label issues?
+
+Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/gatsbyjs/teams/maintainers) can label issues.
+
+You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/docs/how-to-contribute/) to get started.
+
+## How to label an issue
+
+Ideally every issue should have a single `🏷 type:` label applied to it. Optionally a `status: ` label, or other labels may also be applied.
+
+Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels).
+
+The broad steps to labelling an issue are:
+
+- Read an issue
+- Choose the labels that apply to that issue
+- That's it
+
+The rest of this document will describe how to choose the right labels for an issue.
+
+### Find an issue that you're interested in
+
+Start with [Gatsby's issues list](https://github.com/gatsbyjs/gatsby/issues) and scroll through until you see a recent one that strikes your interest. Alternatively you can view the [list of unlabelled issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+no%3Alabel).
+
+### Read the issue
+
+Read the issue and any comments to understand what the issue is about.
+
+### Choose one `type: ` label
+
+Choose a type label from the _labels_ dropdown to the right-hand side of the issue.
+
+!(GitHub label dropdown)(./images/github-label-list.png)
+
+You can check through the [label descriptions](https://github.com/gatsbyjs/gatsby/issues/labels) for more information on each one.
+
+The most common type of issue is `type: question or discussion`, typically you can apply this to issues that are open-ended or have no clear next step.
```

Comment:
This is kind of tangential, but are these the issues we eventually want to move _off_ the issue queue? My feeling is that if an issue won't result in changes to the Gatsby core, it's probably better for a support channel, not GitHub issues.

Reply:
Agreed, but we don't have a specific support channel set up for this yet.